wasmplugin/plugin.go

```diff
@@ -597,6 +598,7 @@ func (ctx *httpContext) OnHttpStreamDone() {
 			// phase that still need to be executed. If they haven't been executed yet, and there has not been a previous
 			// interruption, now is the time.
 			if !ctx.processedResponseBody {
+				ctx.logger.Warn().Msg("Running ProcessResponseBody in OnHttpStreamDone, triggered actions will not be enforced from now on")
```

Comment:
Since this is an expected operation we can use `Info` log level or debug. Also could you ellaborate more and tell that it will trigger another log from coraza (if that is the case?)

Reply:
I went for the `Warn` log level because I was not happy about a case like the following:
Request:
```
curl -I 'http://localhost:8080/status/406'
```
Rule:
```
"SecRuleEngine On",
"SecRule REQUEST_URI \"@rx /status\" \"id:101,phase:4,log,t:lowercase,deny\""
```
Logs:
```
wasm log coraza-filter my_vm_id: [client "172.19.0.1"] Coraza: Warning.  [file ""] [line "986"] [id "101"] [rev ""] [msg ""] [data ""] [severity "emergency"] [ver ""] [maturity "0"] [accuracy "0"] [hostname "172.19.0.2"] [uri "/status/406"] [unique_id "RPLGaNMdNlLPGDUBjpd"]
wasm log coraza-filter my_vm_id: Finished tx_id="cDBqsDMNnWgNYyZSjrp" context_id=3
```

It seems like the rule worked properly, but the disruptive action was not enforced in the end.


Moved to `Info`.